README.md

```diff
@@ -49,6 +49,7 @@ myChart
 | <b>topMargin</b>([<i>number</i>]) | Getter/setter for the chart's top margin, which contains the color legend. | 26 |
 | <b>bottomMargin</b>([<i>number</i>]) | Getter/setter for the chart's bottom margin, which contains the time axis labels. | 30 |
 | <b>useUtc</b>([<i>boolean</i>]) | Getter/setter for whether to display time in the local time zone (`false`) or in UTC (`true`). | false |
+| <b>showToday</b>([<i>boolean</i>]) | Getter/setter for whether to display today as a vertical line | false |
```

Comment:
Perhaps would be useful to generalise it to mark any timestamp, instead of just the now? Something like:
`.showTimeMarker(<date object>)`. If `falsy` (default) no marker would be shown. Wdyt?